packages/injectable/src/getInjectable/getInjectable.js

```diff
@@ -1,5 +1,12 @@
 import identity from 'lodash/fp/identity';
+import lifecycleEnum from '../dependency-injection-container/lifecycleEnum';
 
 // Note: this function exists only for typed presence in TypeScript.
 // It has little purpose in JavaScript.
-export default identity;
+export default function getInjectable(rawInjectable) {
+  if (!rawInjectable.lifecycle) {
+    rawInjectable.lifecycle = lifecycleEnum.singleton;
+  }
```

Comment:
Why do we need this? The public product of `getInjectable()` should not need this information?

Reply:
I did this so that the typings would be actually correct. They accept an optional `lifecycle` param but produce a required one.